index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <!-- <link rel="icon" type="image/svg+xml" href="/vite.svg" /> -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>survive the monster</title>
+  </head>
+  <body>
+    <header>
+      <p>Demo</p>
+    </header>
+    <main>
+      <canvas id="gameCanvas"></canvas>
+    </main>
+    <footer>
+      <p>Zombie</p>
+    </footer>
+    <script type="module" src="/main.js"></script>
```

Comment:
**suggestion**: Ensure that the path to the main.js script is correct and accessible when deploying the project.